grug/testing/tests/reply.rs

```diff
@@ -131,7 +131,7 @@ mod replier {
     pub fn reply(ctx: SudoCtx, msg: ReplyMsg, res: SubMsgResult) -> StdResult<Response> {
         let msg = match (res, msg) {
             (GenericResult::Err(wee), ReplyMsg::Fail(execute_msg)) => {
-                println!("replying with error: {}", wee);
+                println!("replying with error: {wee}");
```

Comment:
I think this variable should be called `err` instead of `wee`, should be a type, but `wee` sounds funny so I'll keep it.